client/starwhale/api/_impl/job.py

```diff
@@ -319,3 +402,57 @@ def generate_jobs_yaml(
         ),
         parents=True,
     )
+
+
+class HanderInput(ABC):
+    def __init__(self, required: bool = False) -> None:
+        self.required = required
+
+    @abstractmethod
+    def parse(self, user_input: t.Any) -> t.Any:
+        ...
+
+
+class ListInput(HanderInput):
+    def __init__(self, member_type: t.Any, required: bool = False) -> None:
+        super().__init__(required)
+        self.member_type = member_type
+
+    def parse(self, user_input: t.List) -> t.Any:
+        if not user_input:
+            return user_input
+        if isinstance(self.member_type, HanderInput):
```

Comment:
Example:
```
class Base():
   ...
class A(Base):
  ...
class B(Base):
  ...
```
when  member_type is `B()`, it will rase `TypeError` because of `issubclass() arg 1 must be a class`

Reply:
done